src/unifiedreport/agent/unifiedreport.php

```diff
@@ -791,6 +792,12 @@ private function writeReport($contents, $uploadId, $groupId, $userId)
 
     /* Display Do not use license files */
     $heading = "Do not use Files";
+    // adding an internal bookmark
+    $rightColStyleBlue = array("size" => 11, "color" => "0000A0", 'underline' => 'single');
```

Comment:
Can you please change the variable name?

Reply:
Done. Thank you